cloud/aws/services/ec2/instances.go

```diff
@@ -120,6 +121,35 @@ func (s *Service) CreateOrGetMachine(machine *clusterv1.Machine, status *v1alpha
 	return s.CreateInstance(machine, config, clusterStatus)
 }
 
+func (s *Service) runInstance(i *v1alpha1.Instance) (*v1alpha1.Instance, error) {
+	input := &ec2.RunInstancesInput{
+		InstanceType: aws.String(i.Type),
+		SubnetId:     aws.String(i.SubnetID),
+		ImageId:      aws.String(i.ImageID),
+		KeyName:      i.KeyName,
+		EbsOptimized: i.EBSOptimized,
```

Comment:
looks like this isn't passed through so it will always be nil

Reply:
Yeah I think that's fine for now, we might want to give the option later on